dev/sparktestsupport/modules.py

```diff
@@ -568,6 +558,15 @@ def __hash__(self):
     ]
 )
 
+# TODO: Skip hive-thriftserver module for hadoop-3.2. remove this once hadoop-3.2 support it
+if os.environ.get("AMPLAB_JENKINS"):
+    hadoop_version = os.environ.get("AMPLAB_JENKINS_BUILD_PROFILE", "hadoop2.7")
+else:
+    hadoop_version = os.environ.get("HADOOP_PROFILE", "hadoop2.7")
+if hadoop_version == "hadoop3.2":
+    print("[info] Skip unsupported module:", "hive-thriftserver")
+    all_modules = [m for m in all_modules if m.name != "hive-thriftserver"]
```

Comment:
Similarily, I tested `all_modules`. This is just to move the codes to remove later in single place.

Reply:
This just does the same thing as before, right?

Reply:
yea I just moved.

Reply:
Got it. Thank you.